lib/index.js

```diff
@@ -3611,6 +3611,78 @@ Stream.prototype.merge = function () {
 exposeMethod('merge');
 
 /**
+ * Takes a Stream of Streams and merges their values and errors into a
+ * single new Stream, limitting the number of unpaused streams that can
+ * running at any one time.
+ *
+ * Note that no guarantee is made with respect to the order in which
+ * values for each stream end up in the merged stream. Values in the
+ * merged stream will, however, respect the order they were emitted from
+ * their respective streams.
+ *
+ * @id mergeLimit
+ * @section Higher-order Streams
+ * @name Stream.mergeLimit()
```

Comment:
Call this `Stream.mergeLimit(n)`. You're also missing a description of the parameter `n`.